backend/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:17-oracle
+ARG JAR_FILE=target/*.jar
+COPY ${JAR_FILE} app.jar
```

Comment:
I think this right now copies a previously created JAR file to Docker image right? Can be move the build step into the container so that frontend team can easily build the image and run the backend application on their local.

Reply:
created a new multistage building Dockerfile